tests/test_pe_source.py

```diff
@@ -19,6 +19,13 @@
     "critical",
 )
 
+# Setup logging to file
+logging.basicConfig(
+    filename="pe_reports_Logging.log",
```

Comment:
How come you are not using the `CENTRAL_LOGGING_FILE` variable here? Also why are you using `filemode="a"` and different formatting options in `src/pe_reports/__init__.py`, but not here?

Reply:
@dav3r 
I updated the script to use the CENTRAL_LOGGING_FILE, made the formatting all the same per commit [4a578f4](https://github.com/cisagov/pe-reports/pull/169/commits/4a578f4c6b35f5360937eb39c88999b363650a55)

Reply:
@dav3r 
I have updated the logging filename and formatting to match throughout the project per comit [7ccf5c9](https://github.com/cisagov/pe-reports/pull/169/commits/7ccf5c9c137ff89f5102120ec00d4ba80ff2c71d) I use the filemode='a' to append the file and as I understand this is what the mode that we want so that we have a continually growing log file. Do you have a recommendation as to another filemode that we should be using?

Reply:
👍  (see https://github.com/cisagov/pe-reports/pull/169#discussion_r952796014 for response).